openquake.cfg

```diff
@@ -0,0 +1,80 @@
+# Copyright (C) 2010-2018 GEM Foundation
```

Comment:
I wonder if this file should be at top-level or close to the openquake.cfg that we have already in openquake/engine; also, this is an example of cluster configuration file, perhaps it should be called openquake-cluster.cfg.
However, I have no strong opinion, so go with whatever makes the most sense to you.

Reply:
Ops, this is NOT the engine repository. Please disregard the commen above.

Reply:
That's not a bad idea at all. I'll do it.

Reply:
My idea is: move this file to `oq-engine/openquake/engine/openquake.cfg.cluster-sample`, then check out `oq-engine` as submodule of this repo and use that file in the docker.

It's also useful for people looking at a cluster cfg.

Reply:
Finxed in https://github.com/gem/oq-engine/pull/4313